### DISCUSSION TITLE
Help on the documentation side

### DISCUSSION THREAD

**Discussion by griffon1115** [acheong08/ChatGPT#139] [General]
Hi, I'm relatively new to GitHub, but would love to contribute on the documentation/writing side what are some things you need done?

**Comment by landa125** [comment]
## Hello, I would like to help also with the documentation/writing side. Currently, work as an IT tech, and I do most of the documentation for our app tutorial and system tutorial for our clients, which consist of 50+ users. Feel free to reach out kan125code@gmail.com

**Comment by acheong08** [comment]
@landa125 @griffon1115 The developer documentation could be worked on. If you don't mind, you can fork the repository and write some documentation under the wiki folder in markdown. I will copy to the GitHub wiki after merging the pull requests.

Thank you!

**Reply by acheong08** [reply]
You can use the current wiki as a base. Some of it is slightly outdated.
